### DISCUSSION TITLE
After config file change (kex_algo change) getting connection refused

### DISCUSSION THREAD

**Discussion by Jcoetsee** [drakkan/sftpgo#779] [Q&A] [answered]
Hi there, I wanted to just direct your attention to this thread where you provided a solution to disable specifically SHA1 algorithms via the config file.

https://github.com/drakkan/sftpgo/discussions/749

It would appear that if we make this change in the config file and restart services, and we attempt to connect to the SFTP server from a windows based client such as FileZilla or WinSCP we are getting "Connection refused" errors.

**Comment by drakkan** [comment]
Strange, it works for me, tested with WinSCP 5.17.10 and Filezilla 3.58.0.

Connection refused generally means that the service is not running so the issue could be:

1) you did a mistake editing the configuration file and the service does not start. You can check the logs
2) you have a firewall that blocks access

**Reply by Jcoetsee** [reply]
@drakkan did you add the line entry the exact same way as you recommended me to add it in the config file in that other thread ?
Would you mind posting that here again?

**Comment by Jcoetsee** [comment, marked as answer]
This thread can be closed, the issue was on my side with the config file

**Reply by drakkan** [reply]
Thanks for the confirmation
